File: URI/2596.py
from math import pow

N = int(input())

for i in range(N):
	T = int(input())

	aux = 1; total = T
	for h in range(1,T+1):
		if h == pow(aux,2):
			aux += 1
			total -= 1

	print(total)

File: URI/2596.c
#include<stdio.h>
#include<math.h>

int main(){
	int N,T;
	scanf("%d",&N);

	for(int i=0; i<N; i++){
		scanf("%d",&T);
		int aux = 1,total = T;
		for(int h=1; h<=T; h++){
			if(h == pow(aux,2)){
				total--;
				aux++;
			}
		}
		printf("%d\n",total);
	}

	return 0;
}